src/Symfony/Component/Finder/Finder.php

```diff
@@ -540,9 +540,9 @@ public function in($dirs)
 
         foreach ((array) $dirs as $dir) {
             if (is_dir($dir)) {
-                $resolvedDirs[] = $dir;
+                $resolvedDirs[] = rtrim($dir, '/\\');
```

Comment:
should be `rtrim($dir, '/'.\DIRECTORY_SEPARATOR);`

Reply:
`\DIRECTORY_SEPARATOR` can be `'/'` or `'\'`

`'/'.\DIRECTORY_SEPARATOR` would be literal `//` on *nix systems.

I can change that of course, just asking myself what the benefit would be

Reply:
the benefit would be that on *nix, `\` should not be trimmed

Reply:
good point, thanks.